### DISCUSSION TITLE
Adjust the size of the viewer for reveal.js

### DISCUSSION THREAD

**Discussion by fretwurst** [posit-dev/positron#4836] [Feedback] [answered]
Is it possible to decouple the viewer? I enjoy creating reveal.js slides with Quarto in Positron. Currently, the viewer is more in portrait mode, which isn't ideal for reveal.js slides. It would be great to have a solution where the preview format of the slides can be adjusted, with blank, grey strips at the top and bottom, or something similar.
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/849ed7f8-5f13-4d4d-846d-45bbbe002781">

**Comment by jmcphers** [comment]
Yes, we're working on this! https://github.com/posit-dev/positron/pull/4808

**Reply by fretwurst** [reply]
Awesome! I hadn't realized that this also works for the preview of the reveal slides.

**Reply by fretwurst** [reply]
If I understand correctly, #4808 has already been merged into the prerelease. However, I can't figure out how to change the narrow portrait format of the previews for reveal.js slides to, for example, a 16:10 format (or perhaps even just two slides stacked vertically). Do I need to adjust any settings somewhere to achieve this?
<img width="1919" alt="image" src="https://github.com/user-attachments/assets/9bc34982-947d-4344-bd43-873c28b0741f">

**Reply by juliasilge** [reply, marked as answer]
Ah yep, I can understand the confusion. What #4808 allows you to do is pop your content from the Viewer pane into an editor tab:

![reveal-slides](https://github.com/user-attachments/assets/de00c3a9-9d8a-4f2f-97cd-f7578b2a200e)

This gives you more room as well as a more natural aspect ratio for viewing slides. Unfortunately, it does _not_ give you a workflow where you can see both your rendered slides and `.qmd` at the same time. For a setup like that, you might consider dragging the Viewer pane into what's called "Session" like this:

![viewer-in-session](https://github.com/user-attachments/assets/33da5c32-9101-46c3-b917-a6454281648c)

**Reply by fretwurst** [reply]
That was very helpful. I haven’t figured out yet how to pop the content from the Viewer pane into an editor, but I did figure out how to combine the viewer with the PLOTS. Since I had to experiment a bit for that, I recorded my process in case someone else is interested. :-)

https://github.com/user-attachments/assets/e85882d1-9ea5-4f07-8761-d800f494c81d
